survival4d/nn/torch/__init__.py

```diff
@@ -31,7 +31,10 @@ def train_nn(xtr, ytr, batch_size, n_epochs, model_name, lr_exp, alpha, weight_d
 
     # Arrange data into minibatches (based on specified batch size), and within each minibatch,
     # sort in descending order of survival/censoring time (see explanation of Cox PH loss function definition)
-    # X_tr, E_tr, TM_tr, _ = sort4minibatches(X_tr, E_tr, TM_tr, batch_size)
+    sort_idx = torch.argsort(torch.as_tensor(TM_tr), descending=True, dim=0)
+    TM_tr = TM_tr[sort_idx]
+    X_tr = X_tr[sort_idx, :]
+    E_tr = E_tr[sort_idx, :]
```

Comment:
we don't need to sort at the beginning anymore, since sorting is done for each batch